File: gerenciador_tarefas/lib/models/usuario.dart
class Usuario {
  final int? id;
  final String nome;
  final String email;
  final String ra;
  final String serie;

  Usuario({this.id, required this.nome, required this.email, required this.ra, required this.serie});

  factory Usuario.fromJson(Map<String, dynamic> json) => Usuario(
        id: json['id'],
        nome: json['nome'] ?? '',
        email: json['email'] ?? '',
        ra: json['rA'] ?? json['ra'] ?? '',
        serie: json['serie'] ?? '',
      );

  Map<String, dynamic> toJson() => {
        if (id != null) 'id': id,
        'nome': nome,
        'email': email,
        'rA': ra,
        'serie': serie,
      };
}

File: src/main/java/org/example/model/Usuario.java
package org.example.model;
import jakarta.persistence.*;

@Entity
public class Usuario {

    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private Long id;
    private String nome;
    private String email;
    private String RA;
    private String serie;

    public Long getId() {
        return id;
    }

    public void setId(Long id) {
        this.id = id;
    }

    public String getNome() {
        return nome;
    }

    public void setNome(String nome) {
        this.nome = nome;
    }

    public String getEmail() {
        return email;
    }

    public void setEmail(String email) {
        this.email = email;
    }

    public String getRA() {
        return RA;
    }

    public void setRA(String RA) {
        this.RA = RA;
    }

    public String getSerie() {
        return serie;
    }

    public void setSerie(String serie) {
        this.serie = serie;
    }
}
